libekb.H

```diff
@@ -43,6 +43,12 @@ struct HWCallout
     // in POWER cec device tree
     std::vector<uint8_t> target_entity_path;
 
+    // To indicate, a planar callout is required for this error.
+    bool isPlanarCallout;
```

Comment:
It is better to define new  clock structure and add all the required info filled in the structure.
This will have impact in phosphor-logging/openpower-proc-control code.

Reply:
i didnt get, whether creating new structure will have impact phosphor-logging/openpower-proc-control code
or current code will have impact?